### ISSUE TITLE
Icon not rendered for ui.stat

### ISSUE THREAD

**Issue by mtanco** [h2oai/wave#1430]
#### Wave SDK Version, OS
0.21.0 - tested locally OSK and also in the Editable Tour in Internal

#### Actual behavior
No waving blue hand next to 100 (grey hand to show this is a valid icon
![image](https://user-images.githubusercontent.com/4793561/169105634-803bde8d-8653-4cd5-9ddd-30c3d8d66642.png)

#### Expected behavior

(A clear and concise description of what you expected to happen.)

#### Steps To Reproduce
```python
from h2o_wave import main, app, Q, ui


@app('/')
async def serve(q: Q):
    q.page['example'] =  ui.tall_stats_card(
        box='1 1 2 4',
        items=[
            ui.stat(label='Fake', value='100', caption='test', icon='WavingHand', icon_color='blue'),
            ui.stat(label='PARAMETER NAME', value='125%'),
            ui.stat(label='PARAMETER NAME', value='578 Users'),
            ui.stat(label='PARAMETER NAME', value='25K')
        ]
    )
    q.page["header"] = ui.header_card(
        box="1 1 -1 1",
        title="",
        subtitle="",
        icon="WavingHand"
    )
    
    await q.page.save()
```

**Comment by mturoci** [comment]
Hm.. although the API allows for `icon`, there is no accompanying implementation. According to https://github.com/h2oai/wave/issues/1067, the proposed design does not include icons, but I would say they can be beneficial. The question is, where should those be placed though.

**Comment by mtanco** [comment]
I'm personally okay with just getting rid of the icon... I just saw something that wasn't working. 

But if we keep it probably before the Value

**Comment by mturoci** [comment]
Agree, however, we cannot get rid of APIwise so I will add a comment to `tall_stats` that icons are not yet supported until we figure out the proper designs.

Done in ed54c4183ac49287f46f76c2e68cdb9fff943c79

**Comment by azim-b** [comment]
it'd be useful to have this feature / icons in a tall stats card
